pylint_odoo/misc.py

```diff
@@ -172,6 +166,55 @@ def _check_{NAME_KEY}(self, module_path)
                         node.file = node_file_original
                         node.lineno = node_lineno_original
 
+    def visit_module(self, node):
+        self.wrapper_visit_module(node)
+
+    def add_message(self, *args, **kwargs):
+        version = (self.manifest_dict.get('version')
+                   if isinstance(self.manifest_dict, dict) else '')
+        match = re.match(r"(?P<version>\d+.\d+)\.\d+\.\d+\.\d+$", version)
```

Comment:
Here you can re-use global variables: `DFTL_MANIFEST_VERSION_FORMAT.format(valid_odoo_versions=version)`

Reply:
In fact, if the module don't have the good version standard the second criteria is use the `valid_odoo_versions` from config parameter.

I mean,
```python
valid_version = self.conf.valid_odoo_versions  # from pylint parameter
if match and len(name_check) >= 1:
    valid_version = match.group('version')  # from manifest
```

Reply:
I try to used the regular expresion defined in `DFTL_MANIFEST_VERSION_FORMAT` but this regular expresion not work as we need

```python
(Pdb) re.match(r"(?P<version>\d+.\d+)\.\d+\.\d+\.\d+$", '11.0.1.0.0').groups()
('11.0',)
(Pdb) re.match(r"({valid_odoo_versions})\.\d+\.\d+\.\d+$", '11.0.1.0.0')
```

Reply:
I try to used the config to get the default default version but not working

```python
(Pdb) self.conf.valid_odoo_versions
*** AttributeError: 'FormatChecker' object has no attribute 'conf'
```